CMake/add_project_kwiver.cmake

```diff
@@ -73,14 +73,18 @@ ExternalProject_Add(kwiver
     -DKWIVER_ENABLE_DARKNET:BOOL=${VIAME_ENABLE_YOLO}
     -DKWIVER_ENABLE_C_BINDINGS:BOOL=${VIAME_ENABLE_PYTHON}
     -DKWIVER_ENABLE_PYTHON:BOOL=${VIAME_ENABLE_PYTHON}
+    -DKWIVER_SYMLINK_PYTHON:BOOL=${VIAME_SYMLINK_PYTHON}
     -DKWIVER_ENABLE_CUDA:BOOL=${VIAME_ENABLE_CUDA}
     -DKWIVER_ENABLE_DOCS:BOOL=${VIAME_ENABLE_DOCS}
     -DKWIVER_INSTALL_DOCS:BOOL=${VIAME_ENABLE_DOCS}
     -DKWIVER_ENABLE_TRACK_ORACLE:BOOL=${VIAME_ENABLE_KWANT}
+    -D KWIVER_ENABLE_TESTS:BOOL=On
```

Comment:
The space between D and K may cause a problem

Reply:
How so? I've always used a space between -D and the variable when writing scripts that invoke cmake, but after looking at all the internal code I see the overwhelming pattern is to not have the space. I'm curious under what conditions would the space cause problems?

Reply:
If nothing else, we may as well be self-consistent.